tests/Makefile

```diff
@@ -0,0 +1,29 @@
+all: lint unit_tests
+
+lint:
```

Comment:
@smarnach We use prospector for OpenCraft IM - this would allow to have the same conventions/configuration for the linters on both sides.

Same thing for the `Makefile` rules - given that this is part of the same project, could you use the same rule names when appropriate?